script.js

```diff
@@ -1,4 +1,80 @@
+//generate a number from 1-3
+// 1 = scissors, 2 = stone, 3 = paper
+// programme outputs either scissors, paper or stone
+// if user inputs scissors and programme outputs scissors = draw
+// if user inputs scissors and programme outputs paper = win
+// if user inputs scissors and programme outputs stone = lose
+// if user inputs paper and programme outputs paper = draw
+// if user inputs paper and programme outputs scissors = lose
+// if user inputs paper and programme outputs stone = win
+// if user inputs stone and programme outputs stone = draw
+// if user inputs stone and programme outputs paper = lose
+// if user inputs stone and programme outputs scissors = win
```

Comment:
I like this very much! Very clear display of pseudo-code logic! Excellent work! Something I would have done differently would be to group all the win and all the losses together to make it clearer to me but if this works for you, that's great!